src/test/java/olivero/parsers/ParserTest.java

```diff
@@ -0,0 +1,232 @@
+package olivero.parsers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.Test;
+
+import olivero.commands.ByeCommand;
+import olivero.commands.DeadlineCommand;
+import olivero.commands.DeleteCommand;
+import olivero.commands.EventCommand;
+import olivero.commands.ListCommand;
+import olivero.commands.MarkCommand;
+import olivero.commands.ToDoCommand;
+import olivero.commands.UnMarkCommand;
+import olivero.exceptions.CommandParseException;
+
+
+/**
+ * Tests for the Parser member SUTs.
+ */
+public class ParserTest {
+
+    private static final String MESSAGE_EXPECTED_INVALID_INTEGER = "Did you pass "
+            + "in a valid integer? Your input: %s";
+
+    @Test
+    public void parse_blankArguments_exceptionThrown() {
+        String expected = "W-WHAT?! "
+                + "I do not understand what you just said :(";
```

Comment:
```suggestion
        String expected = Responses.RESPONSE_UNKNOWN_COMMAND
```

Is there any reason the expected responses are not imported from common.Responses? It would help if you had to make changes/add to the responses along the way (less lines to edit)

Reply:
This helps to keep the testing code independent of the source code, to prevent false negatives